spec/unit/intent.spec.js

```diff
@@ -19,7 +19,7 @@ describe("Intent", function() {
         var clientFields = [
             "credentials", "joinRoom", "invite", "leave", "ban", "unban",
             "kick", "getStateEvent", "setPowerLevel", "sendTyping", "sendEvent",
-            "sendStateEvent", "setDisplayName", "setAvatarUrl"
+            "sendStateEvent", "setDisplayName", "setAvatarUrl", "getRoom"
```

Comment:
where is getRoom used?

Reply:
Nowhere – it's a leftover from the old implementation.

Reply:
Can probably drop this then :)